src/sidebar/services/test/tags-test.js

```diff
@@ -56,6 +56,11 @@ describe('TagsService', () => {
         count: 1,
         updated: stamp,
       },
+      ინგლისური: { // "English" in Georgian
+        text: 'ინგლისური',
+        count: 1,
+        updated: stamp,
+      },
```

Comment:
Aside from this new test case, all of the tags consist only of letters and spaces. This misses out many scenarios involving even just ASCII characters.